src/PowerShellEditorServices/Services/PowerShell/Host/PsesInternalHost.cs

```diff
@@ -1085,7 +1098,7 @@ private void OnPowerShellIdle(CancellationToken idleCancellationToken)
                     // TODO: This may not be a PowerShell task, so ideally we can differentiate that here.
```

Comment:
Maybe we should resolve this todo and only execute it if it's a PowerShell task?